docs/source/quickstart/local-subgraphs.mdx

```diff
@@ -47,7 +47,7 @@ And that's it! You can now perform all of the same subgraph setup on your _own_
 
 In a federated graph, each subgraph's schema can define or reference types and fields that are present in _another_ subgraph's schema.
 
-Consider the `Product` type in our composed supergraph schema from [Part 3](./quickstart/local-composition/#2-perform-composition). This type includes the following fields:
+Consider the `Product` type in our composed supergraph schema from [Part 3](./local-composition/#2-perform-composition). This type includes the following fields:
 
 ```graphql
 type Product
```

Comment:
I believe the `extension: true` wouldn't be there on the `@join__type(graph: REVIEWS, key: "id", extension: true)` since the examples don't use `extend type` anymore.